mentoring/mentoring.py

```diff
@@ -155,11 +148,17 @@ def is_default_xml_content(value):
         scope=Scope.preferences
     )
 
+    editable_fields = (
+        'mode', 'followed_by', 'max_attempts', 'enforce_dependency',
+        'display_submit', 'weight', 'display_name',
+    )
     icon_class = 'problem'
     has_score = True
     has_children = True
 
-    FLOATING_BLOCKS = (TitleBlock, MentoringMessageBlock, SharedHeaderBlock)
+    def _(self, text):
```

Comment:
Yes to both. I will be doing a pass through this whole repo and internationalizing all the things as part of OC-541